src/vue/modules/create-asset-sale/store/index.js

```diff
@@ -0,0 +1,26 @@
+import { types } from './types'
+import { api } from '../_api'
+import { AssetRecord } from '../wrappers/asset.record'
+
+export const actions = {
+  async [types.LOAD_KV_KYC_REQUIRED] ({ commit, getters }) {
+    const { data } = await api().get(`v3/key_values/asset_type:kyc_required`)
```

Comment:
Move v3 to the const as in the other modules

Reply:
ok